### ISSUE TITLE
memcalc branch fails to run compile script locally

### ISSUE THREAD

**Issue by glyn** [cloudfoundry/java-buildpack#375]
At 0f39e3eeed24bea46e1877a, I see this:
```
$ bin/compile ~/Dropbox/WAR-Files/axis2exploded/ $TMPDIR
-----> Java Buildpack Version: 0f39e3e | https://github.com/cloudfoundry/java-buildpack.git#0f39e3e
-----> Downloading Open Jdk JRE 1.8.0_121 from https://java-buildpack.cloudfoundry.org/openjdk/mountainlion/x86_64/openjdk-1.8.0_121.tar.gz (found in cache)
       Expanding Open Jdk JRE to .java-buildpack/open_jdk_jre (0.6s)
-----> Downloading Open JDK Like Memory Calculator 3.0.1_RELEASE from https://java-buildpack.cloudfoundry.org/memory-calculator/mountainlion/x86_64/memory-calculator-3.0.1_RELEASE.tar.gz (found in cache)
[Buildpack]                      ERROR Compile failed with exception #<Zip::Error: File /Users/gnormington/Dropbox/WAR-Files/axis2exploded/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/container_certificate_trust_store/container_certificate_trust_store-2.0.0_RELEASE.jar not found>
File /Users/gnormington/Dropbox/WAR-Files/axis2exploded/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/tomcat/webapps/ROOT/.java-buildpack/container_certificate_trust_store/container_certificate_trust_store-2.0.0_RELEASE.jar not found
```

**Comment by nebhale** [comment]
@glyn It's likely because you aren't running the commend in the context of `bundler`.  Remember you'll need to run `bundle exec bin/compile ...` to ensure that you've got everything configured properly.  You'll also likely need to `bundle install` for that to work properly.  Since it's likely been a while since you've updated dependencies `gem install bundler` is probably useful to get onto a modern version of that as well.

**Comment by nebhale** [comment]
Resolved by c45b7f3
